File: Codes/c_Run_Tools/BoxWrench/_vendor/General/RotMat.py
import math
import numpy as np

def RotMat(x,y,z):
    x=math.radians(x)
    y=math.radians(y)
    z=math.radians(z)
    Rx=np.array([[1, 0, 0], [0, math.cos(x), -math.sin(x)],[0, math.sin(x), math.cos(x)]])
    Ry=np.array([[math.cos(y), 0, math.sin(y)], [0, 1, 0],[-math.sin(y), 0, math.cos(y)]])
    Rz=np.array([[math.cos(z), -math.sin(z), 0], [math.sin(z), math.cos(z), 0],[0, 0, 1]])
    return(np.dot(Rx, np.dot(Ry, Rz)))

File: a_Get Exp Data/RotMat.m
function R = RotMat(x,y,z)
x=x*pi/180;
y=y*pi/180;
z=z*pi/180;
Rx=[1 0 0;0 cos(x) -sin(x);0 sin(x) cos(x)];
Ry=[cos(y) 0 sin(y);0 1 0;-sin(y) 0 cos(y)];
Rz=[cos(z) -sin(z) 0;sin(z) cos(z) 0;0 0 1];
R=Rx*Ry*Rz;
end